### ISSUE TITLE
Error installing gem with version '~> 0.4.0'

### ISSUE THREAD

**Issue by sic-f** [renchap/webpacker-react#56]
I get this: 
`Could not find gem 'webpacker-react (~> 0.4.0)' in any of the gem sources listed in your Gemfile.`

**Comment by BookOfGreg** [comment]
https://github.com/renchap/webpacker-react/releases

Looks like 0.4 hasn't been released yet.

**Comment by vmarquet** [comment]
Yes, it has not been pushed to rubygems yet: https://rubygems.org/gems/webpacker-react/

**Comment by mayrsascha** [comment]
@renchap Is it possible to release 0.4 already as it's stated [in the version file](https://github.com/renchap/webpacker-react/blob/master/lib/webpacker/react/version.rb)? I got confused by the README as on one hand this library is probably ready to work with Webpack 4 ([source](https://github.com/renchap/webpacker-react#installation)) and on the other hand also not ([source](https://github.com/renchap/webpacker-react#note-about-versions)).

**Comment by renchap** [comment]
Sorry for the delay, it took me a long time to get back to maintaining this gem properly.

I just pushed [version 1.0.0-beta.1](https://github.com/renchap/webpacker-react/releases/tag/v1.0.0.beta.1) to Rubygems & NPM, feel free to test it!
